frontend/src/game.js

```diff
@@ -116,15 +127,17 @@ export class Game {
     // private
     process_click(event) {
         console.log('clicked', event.target.coords);
-
-        this.send_to_backend('click', event.target.coords);
+        if (this.players_action_enabled) {
+            this.send_to_backend('click', event.target.coords);
```

Comment:
I think it would be better to check player_action inside send_to_backend

It seems, that we don't want send any events when frontend is locked. At lest for now. But you have to remember to add this check before every send_to_backend function

Reply:
Great suggestion, bro. I'll fix it now.